framework_tvb/tvb/interfaces/web/controllers/project/project_controller.py

```diff
@@ -338,8 +338,12 @@ def get_datatype_details(self, entity_gid, back_page='null', exclude_tabs=None):
         current_type = datatype_details.data_type
         datatype_gid = datatype_details.gid
         categories = {}
+        no_visualisers = None
         if not entity.invalid:
-            categories = self.flow_service.get_launchable_algorithms(datatype_gid)
+            try:
+                categories = self.flow_service.get_launchable_algorithms(datatype_gid)
+            except Exception as excep:
+                no_visualisers = 'Not all operations could be loaded for this input DataType. Contact the admin to check the logs!'
```

Comment:
Now we actually catch too late the exception, thus we will never show any viewer, when we should only omit the ones with problems